form-data/form-data.d.ts

```diff
@@ -8,6 +8,7 @@
 declare module "form-data" {
     class FormData {
         append(key: string, value: any, options?: any): void;
+        entries(): IteratorResult<string,Object>;
```

Comment:
It seems entries only supported by chrome + 50, and yes, it won't work in es5/3. Should I stop adding this?